cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/DefineMissingContainersPreProcessor.java

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.cgmes.conversion;
+
+import com.google.auto.service.AutoService;
+import com.powsybl.cgmes.model.CgmesModel;
+import com.powsybl.cgmes.model.triplestore.CgmesModelTripleStore;
+import com.powsybl.commons.config.PlatformConfig;
+import com.powsybl.commons.datasource.FileDataSource;
+import com.powsybl.commons.reporter.Reporter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Objects;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+/**
+ * @author Luma Zamarreño <zamarrenolm at aia.es>
+ */
+@AutoService(CgmesImportPreProcessor.class)
+public class DefineMissingContainersPreProcessor implements CgmesImportPreProcessor {
```

Comment:
To be discussed: not sure we want this pre-processor in cgmes-conversion. In a new module, cgmes-validation?

Reply:
I prefer the option of using a separate module instead of a different repo, as it would keep closer all the cgmes-related code.

I would like a name that, in addition to "validation", suggests that we could also be "fixing" the model ... cgmes-fixes ? cgmes-completion ?

Reply:
I've moved this pre-processor to a new module named cmges-completion.